frontend/src/app/person/person.component.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('PersonComponent', () => {
     abroadFamilySituation: null
   };
 
-  const activatedRoute = {
+  const activatedRoute: any = {
```

Comment:
I would love to, but it doesn't compile. I don't know exactly why. I always thought of 'as Foo' to mean "yeah, I know it's not that type, but don't bother", but that doesn't actually work that way:

```
ERROR in /Users/jb/projects/globe42/frontend/src/app/person/person.component.spec.ts (45,26): Type '{ parent: { snapshot: { data: { person: PersonModel; }; }; }; }' cannot be converted to type 'ActivatedRoute'.
  Property 'url' is missing in type '{ parent: { snapshot: { data: { person: PersonModel; }; }; }; }'.
```